janus_core/calculations/geom_opt.py

```diff
@@ -167,11 +170,13 @@ def __init__(
         angle_tolerance
             Angle precision for spglib symmetry determination, in degrees. Default is
             -1.0, which means an internally optimized routine is used to judge symmetry.
+        filter_class
+            Filter class, or name of class from ase.filters to wrap around atoms.
+            Default is `FrechetCellFilter`.
         filter_func
-            Filter function, or name of function from ase.filters to apply constraints
-            to atoms. Default is `FrechetCellFilter`.
+            Deprecated. Please use `filter_class`.
```

Comment:
Numpydoc provides an [`Other Parameters`](https://numpydoc.readthedocs.io/en/latest/format.html#other-parameters) block to which we could relegate the deprecated args?

Reply:
This almost seems to have the effect of making them (`filter_func`, alongside `model_path` if we went down this route) more prominent, since the section still shows up in the hints, but they're no longer next to their replacements, so I think I may leave it as is.